includes/class-logger.php

```diff
@@ -68,19 +54,7 @@ public static function init_logger() {
 	 * @return bool Depending on the enable_logging setting.
 	 */
 	public static function can_log() {
-		if ( ! function_exists( 'wc_get_logger' ) ) {
-			return false;
-		}
-
-		if ( is_null( WC_Payments::get_gateway() ) ) {
```

Comment:
Now that this line is gone, we don't need `use WC_Payments` in the beginning of the file.

Reply:
Removed